scripts/updates/utils/termux_pkg_is_update_needed.sh

```diff
@@ -9,20 +9,19 @@ termux_pkg_is_update_needed() {
 	local CURRENT_VERSION="$1"
 	local LATEST_VERSION="$2"
 
+	# Is this even a validly formatted version number?
+	if ! dpkg --validate-version "${LATEST_VERSION}" &> /dev/null && [[ "${CI-false}" == "true" ]]; then
```

Comment:
Remove '`&& [[ "${CI-false}" == "true" ]]`'.
It should never be allowed to skip error both locally and on CI.
You may or may not want to keep `::warning::`, but I don't see any reason to keep it.